manim/utils/cfg_subcmds.py

```diff
@@ -8,12 +8,12 @@
 """
 import os
 import configparser
+from ast import literal_eval
```

Comment:
Issue: I know that `literal_eval` is safer than `eval`, but I would still only want to use it in case it is 100% absolutely necessary. Can we think of safety issues that using `literal_eval` may raise? For example, can the user write arbitrary python code in the user-wide config file and will that code be executed?

Reply:
> For example, can the user write arbitrary python code in the user-wide config file and will that code be executed?

I'm no security expert, but I don't _think_ this is possible.
Trying some stuff out:

```py
>>> eval("[a for a in range(1,4)]")
[1, 2, 3] #Works for eval

>>> literal_eval("[a for a in range(1,4)]")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 99, in literal_eval
    return _convert(node_or_string)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 98, in _convert
    return _convert_signed_num(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 75, in _convert_signed_num
    return _convert_num(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 66, in _convert_num
    _raise_malformed_node(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 63, in _raise_malformed_node
    raise ValueError(f'malformed node or string: {node!r}')
ValueError: malformed node or string: <_ast.ListComp object at 0x103163790> #Doesn't work for literal_eval
```

So even innocent things like list comprehensions are out.
Trying some `import os` stuff also has a similar story:

```py
>>> eval("__import__('os')")
<module 'os' from '/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/os.py'>
#Works in eval
>>> literal_eval("__import__('os')")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 99, in literal_eval
    return _convert(node_or_string)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 98, in _convert
    return _convert_signed_num(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 75, in _convert_signed_num
    return _convert_num(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 66, in _convert_num
    _raise_malformed_node(node)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/ast.py", line 63, in _raise_malformed_node
    raise ValueError(f'malformed node or string: {node!r}')
ValueError: malformed node or string: <_ast.Call object at 0x103163a30>
# Doesn't work in literal_eval
```
But at the end of the day, it's the users _own_ computer that's going to be impacted, so I highly doubt anyone will try to put malicious code through their own systems...

Reply:
> But at the end of the day, it's the users own computer that's going to be impacted, so I highly doubt anyone will try to put malicious code through their own systems...

Agreed, but if we end up using `eval`, then we should be extremely careful about what goes in the `default.cfg` file.

Ok after seeing your examples and reading up on the documentation of `literal_eval`, I think it's fine to use it.